src/main/java/homework7/Main.java

```diff
@@ -0,0 +1,80 @@
+package homework7;
+
+import homework5.Fruits;
+import homework6.Vitamins;
+import lesson6.Animal;
+
+import java.io.*;
+import java.lang.reflect.Constructor;
+import java.lang.reflect.Method;
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import static homework5.FruitsEnum.*;
+import static homework6.Vitamins.*;
+
+public class Main {
+    public static void main(String[] args) {
+        homework6.Main main = new homework6.Main();
```

Comment:
What sence?

Reply:
It isn't need

Reply:
In homework6.Main is a private method fillFruitList. This instance is here to use it

Reply:
via reflection

Reply:
I redone with Class.forName()

Reply:
but it dont work